play-json/shared/src/test/scala/play/api/libs/json/WritesSharedSpec.scala

```diff
@@ -104,17 +105,11 @@ final class WritesSharedSpec extends AnyWordSpec with Matchers {
 
   "EnumFormat" should {
     import TestEnums.EnumWithCustomNames._
-    import TestEnums.EnumWithDefaultNames._
 
     "serialize correctly enum with custom names" in {
       Json.toJson(customEnum1).mustEqual(JsString("ENUM1"))
       Json.toJson(customEnum2).mustEqual(JsString("ENUM2"))
     }
-
```

Comment:
?

Reply:
Moved to EnumSpec, because of Scala3.js.